notebooks/COS/CalCOS/CalCOS.ipynb

```diff
@@ -8,39 +8,39 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    calcos.calcos(str(datadir/asn_name),
</pre><p>Also, running this line a second or more times will throw a Runtime error. To avoid such a traceback in the output cell we can implement a try, except block and include the error message within the captured file.</p><p> </p><pre class="ql-syntax" spellcheck="false">try:
    calcos.calcos(
        str(datadir/asn_name),
&nbsp; &nbsp; &nbsp; &nbsp; verbosity=2,
&nbsp; &nbsp; &nbsp; &nbsp; outdir=str(outputdir/"calcos_processed_4"))
except RuntimeError as e:
    print("An error occurred:", e)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>